hawkbit-ui/src/main/resources/messages.properties

```diff
@@ -496,6 +498,7 @@ message.key.deleted.or.notAllowed = {0} with key {0} was deleted or you are not
 message.forcedTime.cannotBeEmpty = Forced time can not be empty
 message.forcedTime.missing = Force time is not specified for the time-forced assignment
 message.scheduledTime.cannotBeEmpty = Scheduled time can not be empty
+message.clear.file.upload.queue.confirm = There is still at least one queued file upload pending. By leaving this view the upload queue will be cleared. \nAre you sure you want to leave this view and cancel the pending file upload?
```

Comment:
The text uses the word "cancel". But in the dialog itself the Ok button confirms this text and the cancel button does the opposite. 
Replace "cancel" in the text with something else, e.g. "stop"

Reply:
DONE